src/Symfony/Component/Serializer/Command/DebugCommand.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Dumper;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Exception\DirectoryNotFoundException;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Serializer\Mapping\ClassMetadataInterface;
+use Symfony\Component\Serializer\Mapping\Factory\ClassMetadataFactoryInterface;
+
+/**
+ * A console command to debug Serializer information.
+ *
+ * @author Loïc Frémont <lc.fremont@gmail.com>
+ */
+#[AsCommand(name: 'debug:serializer', description: 'Display serialization information for classes')]
+class DebugCommand extends Command
+{
+    public function __construct(private readonly ClassMetadataFactoryInterface $serializer)
+    {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument('class', InputArgument::REQUIRED, 'A fully qualified class name or a path')
```

Comment:
Is it possible to list all the possible values for shell completion? (last argument of the method).

Reply:
Yes, with the serialization.php from the cache directory, I think it could be possible
https://gist.github.com/loic425/9db59617b5c15fe02111ce7df5d3c63b